src/pages/media.astro

```diff
@@ -0,0 +1,504 @@
+---
+import BaseLayout from "../layouts/BaseLayout.astro";
+import Section from "../components/Section.astro";
+---
+
+<BaseLayout title="Media Kit">
+  <img
+    class="header-image"
+    src={"/img/banners/default-with-bear.svg"}
+    aria-hidden
+  />
+  <main id="main">
+    <Section variant="light" content="list">
+      <h1 class="align-center">JSHeroes Media Kit <br /> Welcome!</h1>
+
+      <div class="container column">
+        <h3>On this page you will find:</h3>
+        <ul>
+          <li>
+            <a href="#about">About JSHeroes</a>
+          </li>
+          <li>
+            <a href="#brand">Logos & Brand</a>
+          </li>
+          <li>
+            <a href="#typeface">Typeface</a>
+          </li>
+          <li>
+            <a href="#image-gallery">Image Gallery</a>
+          </li>
+          <li>
+            <a href="#video-gallery">Video Gallery</a>
+          </li>
+        </ul>
+      </div>
+
+      <div id="about">
+        <h3 class="align-center">About JSHeroes</h3>
+        <p>
+          <b>JSHeroes</b> is a yearly event organised by the local JavaScript community
```

Comment:
I think the recommendation is to use `<strong>` instead of <b> as it is a deprecated tag